addons/l10n_mx/data/account_data.xml

```diff
@@ -30,5 +30,18 @@
         <record id="tax_group_isr_ret_10" model="account.tax.group">
             <field name="name">ISR Retencion 10%</field>
         </record>
+
+        <record id="tax_group_isr" model="account.tax.group">
```

Comment:
Shouldn’t be used this data?
I mean, assign the tax group to tax

Reply:
Could you please check the changes made here addons/l10n_mx/data/account_tax_data.xml  in this PR... Is this what you mean?

@luistorresm

Reply:
Done @moylop260 

Regards